awswrangler/emr.py

```diff
@@ -407,7 +407,7 @@ def create_cluster(  # pylint: disable=too-many-arguments,too-many-locals,unused
     subnet_id: str,
     cluster_name: str = "my-emr-cluster",
     logging_s3_path: Optional[str] = None,
-    emr_release: str = "emr-6.0.0",
+    emr_release: str = "emr-5.36.0",
```

Comment:
Any reason why we would downgrade this from 6.0? I think it would be considered a breaking change perhaps

Reply:
I had initially tried with the latest `emr-6.9.0` but encountered an error:
```bash
 botocore.exceptions.ClientError: An error occurred (ValidationException) when calling the RunJobFlow operation: Classification 'spark-log4j' is not valid for parent classification 'null'.`
```

Because I saw `5.x.x` in a couple of the tests I used latest in the 5 series. After a closer inspection it looks like everything works fine up to `emr-6.7.0` so I've updated default.

Reply:
I remember something similar in https://github.com/aws/aws-sdk-pandas/issues/1938